bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -923,6 +928,41 @@ private void updateDatabaseSchema(String dbPath) throws Exception {
 			releaseSingleUserCaseWriteLock();
 		}
 	}
+	
+	/**
+	 * Get the database schema creation version from database. This must be 
+	 * called after the database upgrades or the tsk_db_info_extended table 
+	 * may not exist.
+	 *
+	 * @throws SQLException
+	 */
+	private void initDBSchemaCreationVersion(CaseDbConnection connection) throws SQLException{
+		
+		Statement statement = null;
+		ResultSet resultSet = null;
+		String createdSchemaMajorVersion = "0";
+		String createdSchemaMinorVersion = "0";
+		acquireSingleUserCaseReadLock();
+		try {
+			statement = connection.createStatement();
+			resultSet = connection.executeQuery(statement, "SELECT name, value FROM tsk_db_info_extended");
+			while(resultSet.next()) {
+				String name = resultSet.getString("name");
+				if (name.equals(CREATION_SCHEMA_MAJOR_VERSION_KEY) || name.equals("CREATED_SCHEMA_MAJOR_VERSION")){
```

Comment:
we should define static final variables for 'CREATED_SCHEMA_MAJOR_VERSION' & 'CREATED_SCHEMA_MINOR_VERSION' and use those, instead of hard-coding the literals here.

Update: Having looked at it further down, if CREATED_* names were just accidental and we are fixing them back to CREATION_*, then I'm fine with hard coding them.

Reply:
That's why I hard coded them.  :-)